.husky/pre-push

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# Check if not running in GitHub Actions
+if [ -z "$CI" ]; then
+    echo "Running pre-push hook in local environment"
+    npm run buttondiff
+    exit 0
```

Comment:
this can be `exit $?` to reuse the exit code from `npm run buttondiff` (which I think will exit 1 for a failure) but wanted to mention this will still allow the push even if the `npm run buttondiff` fails (I think) (which I'm also ok with)

Reply:
Nice catch! 🙏

Reply:
It looks like `git push` doesn't complete when `npm run buttondiff` returns `exit 1`:

<img width="735" alt="Screenshot 2024-03-06 at 2 52 31 PM" src="https://github.com/paypal/paypal-checkout-components/assets/20399044/000399ae-804b-401b-b8c9-85972c20ebdd">

Reply:
yes! I thought this was the desired behavior (if buttondiff fails - don't push)